graph/docu/docu_04graph_02plotutils.cc

```diff
@@ -67,7 +67,9 @@ for (size_t e = 0; e < 2; ++e) {
     .OutputFormat("svg");
 #endif
 if (FLAGS_regenerate_golden_graphs) bricks::FileSystem::WriteStringToFile(result, (std::string("golden/love.") + extensions[e]).c_str());
+#ifndef BRICKS_APPLE // TODO(dkorolev): Figure out how to run this test on Apple.
 if (!e) ASSERT_EQ(result, bricks::FileSystem::ReadFileAsString(std::string("golden/love.") + extensions[e]));
```

Comment:
Aha, you need some console tool installed, right?
Which exactly binary should be called? I suppose it can be easily installed via mac port or brew.